zulipterminal/cli/run.py

```diff
@@ -18,8 +18,20 @@
 from zulipterminal.version import ZT_VERSION
 
 
-LOG_FILENAME = 'zulip-terminal-tracebacks.log'
-logging.basicConfig(filename=LOG_FILENAME, level=logging.DEBUG)
+ZT_LOG_FILENAME = 'zulip-terminal-tracebacks.log'
+API_CALL_LOG_FILENAME = 'API-requests.log'
```

Comment:
We might want to add `zulip-terminal` prefix here as well given that we may end up putting in it `/tmp` later.